docs/how-to/code/create-slice/openssl.yaml

```diff
@@ -1,19 +1,32 @@
 package: openssl
+
+essential:
+  - openssl_copyright
+
 slices:
-    bins:
-        essential:
-          - libc6_libs
-          - libc6_config
-          - libssl3_libs
-          - openssl_config
-        contents:
-            /usr/bin/c_rehash:
-            /usr/bin/openssl:
+  bins:
+    essential:
+      - libc6_config
+      - libc6_libs
+      - libssl3_libs
+      - openssl_config
+    contents:
+      /usr/bin/c_rehash:
+      /usr/bin/openssl:
+
+  config:
+    contents:
+      /etc/ssl/certs/:
+      /etc/ssl/openssl.cnf:
+      /etc/ssl/private/:
+      /usr/lib/ssl/certs:
+      /usr/lib/ssl/openssl.cnf:
+      /usr/lib/ssl/private:
+
+  data:
+    contents:
+      /usr/lib/ssl/cert.pem: {symlink: /etc/ssl/certs/ca-certificates.crt}
```

Comment:
For Ubuntu >= 24.04, the `/usr/lib/ssl/cert.pem -> /etc/ssl/certs/ca-certificates.crt` exists in the openssl package by default. So leave here a comment that the `{symlink: /etc/...}` is only needed for focal and jammy.

Reply:
Thanks. I think I surpassed the line limit with the last suggestion I applied.

Re the comment, I removed it from the original openssl.yaml I copied because I thought the comment would be distracting in the context of demoing slices.